ee/app/ldap-enterprise/server/hooks/beforeSearchAll.js

```diff
@@ -10,4 +8,4 @@ callbacks.add('ldap.beforeSearchAll', (searchParams) => {
 	options.attributes.push('pwdAccountLockedTime');
```

Comment:
It should be considered that microsoft active directory uses `User-Account-Control Attribute Flags` https://ldapwiki.com/wiki/User-Account-Control%20Attribute to control locked out/disabled accounts

Reply:
Thanks @bbrauns but the code doesn't support AD yet, would you mind to contribute this improvement?